src/components.py

```diff
@@ -32,20 +77,39 @@ def create_plot_card(plot_id):
     ], className="mb-6 shadow-sm p-3", fluid=True, style={'height': '100%', 'backgroundColor': 'white'})
 
 
-# Define all the required cards
-cards = dbc.Row([
-    dbc.Col(create_stat_card("Proportion Subscribed", "prop_subscribed"), style={'height': '100%'}),
-    dbc.Col(create_stat_card("Average Contacts (Current Campaign)", "avg_contacts_campaign"), style={'height': '100%'}),
-    dbc.Col(create_stat_card("Average Contacts (Previous Campaigns)", "avg_contacts_before"), style={'height': '100%'}),
-    dbc.Col(create_stat_card("Average Last Contact Duration (Seconds)", "avg_last_contact"), style={'height': '100%'}),
-    dbc.Col(create_stat_card("Subscribed?", "subscribed_summary"), style={'height': '100%'}),
-], style={'height': '95%'})
+cards = html.Div([
+    # Left group - 3 cards
+    html.Div([
+        dbc.Col(create_stat_card([
+            html.Span("Average Contacts", style={'color': '#666666'}),
+            html.Br(),
+            html.Span("CURRENT", style={'color': '#232242', 'fontWeight': 'bold'}),
+            html.Span(" Campaign", style={'color': '#666666'})
+        ], "avg_contacts_campaign")),
+        dbc.Col(create_stat_card([
+            html.Span("Average Contacts", style={'color': '#666666'}),
+            html.Br(),
+            html.Span("PREVIOUS", style={'color': '#232242', 'fontWeight': 'bold'}),
+            html.Span(" Campaigns", style={'color': '#666666'})
+        ], "avg_contacts_before")),
+        dbc.Col(create_stat_card([
+            "Average Last Contact",
+            html.Br(),
+            "Duration",
```

Comment:
```suggestion
            "Duration (Seconds)"
```
I suggest we move the (Seconds) to the same line as Duration:
![image](https://github.com/user-attachments/assets/3402fcbc-c27f-4525-9d2a-3987b63abced)